modules/nw-external-dns-operator.adoc

```diff
@@ -5,13 +5,13 @@
 [id="nw-external-dns-operator_{context}"]
 = External DNS Operator
 
-The External DNS Operator implements the External DNS API from the `olm.openshift.io` API group. The External DNS Operator deploys the `ExternalDNS` using a deployment resource. The ExternalDNS deployment watches the resources such as services and routes in the cluster and updates the external DNS providers.
+The External DNS Operator implements the External DNS API from the `olm.openshift.io` API group. The External DNS Operator deploys the `ExternalDNS` using a deployment resource. The External DNS deployment watches the resources such as services and routes in the cluster and updates the external DNS providers.
```

Comment:
This sentence feels awkward to me: "The External DNS Operator deploys the `ExternalDNS` using a deployment resource."

Can we reword this somehow? I'm not familiar enough with the technical details of this action to suggest anything, but as-is, the sentence feels almost redundant. Does that make sense?

Reply:
It's also anthropomorphizing to say that the External DNS deployment watches the resources IMO.

Reply:
Maybe "updates the relevant external DNS providers"?

Reply:
Updated the paragraph.